raiden/ui/cli.py

```diff
@@ -306,4 +306,4 @@ def run(ctx, **kwargs):
         gevent.signal(signal.SIGINT, event.set)
         event.wait()
 
-        app_.stop()
+        app_.stop(graceful=True)
```

Comment:
What I mean is we may want to test the behaviour of a different, new integration test with the graceful shutdown feature sometime in the future and at that point having it as a boolean fixture would be nice.

If you don't want to add it it's fine we can add it when/if such a situation arises.